map/display-map/gradlew

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env sh
 
+#
```

Comment:
There's an update here to this wrapper file which I'm not sure was intentional?
Most of the wrappers are probably out of date, so might be good to have a separate PR that updates them all?

Reply:
(mind the wrapper.jar file above as well for this sample, it's been changed too)